setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+usage="Usage: ./setup.sh </path/to/venv/>"
```

Comment:
```suggestion
usage="Usage: source ./setup.sh </path/to/venv/>"
```
We should probably have a check to ensure that the script was sourced